package.json

```diff
@@ -10,7 +10,7 @@
     "start:watch": "nodemon --inspect=0.0.0.0 --ext js --legacy-watch index.js",
     "test": "npm run test:unit",
     "test:lint": "standard",
-    "test:unit": "jest --forceExit",
+    "test:unit": "jest --forceExit --detectOpenHandles",
```

Comment:
did you mean to keep the `detectOpenHandles` in? I think this it to assist in diagnosing if there are functions still running

Reply:
I added this at the suggestion of jest and to reduce the warning noise when doing a test run.